README.md

```diff
@@ -9,45 +9,46 @@ for more information.
 
 # Installation and Usage
 
-This code is built using Cython, so you have to run the makefile
-in the directory.  First install the requirements with
+Install using pip
 
 ```
-pip install -r requirements.txt
+pip install git+https://github.com/$repo_location
 ```
 
-and then run
+As with all python packages, it is recommended to create a virtual environment
+when using this project.
+
+# Example
 
 ```
-make
-```
+from mdlp.discretization import MDLP
```

Comment:
Please put back the `>>>`. This allow python doctests to be ran
https://docs.python.org/2/library/doctest.html